docker/releases/build_releases.sh

```diff
@@ -53,50 +57,65 @@ while [ "$1" != "" ]; do
     shift
 done
 
-if [[ ! "$NETWORK" =~ ^mainnet$|^testnet$|^betanet$ ]]
-then
+# Set IMAGE_NAME to docker image name
+case $NETWORK in
+  mainnet)
+    IMAGE_NAME=stable
+    ;;
+  testnet)
+    IMAGE_NAME=testnet
+    ;;
+  betanet)
+    IMAGE_NAME=betanet
+    CHANNEL=beta
+    ;;
+  *)
     echo "$RED_FG[$0]$END_FG_COLOR Network values must be either \`mainnet\`, \`testnet\` or \`betanet\`."
     exit 1
-elif [ "$NETWORK" != "mainnet" ]
-then
-    if [ "$NETWORK" == "betanet" ]
-    then
-        CHANNEL=beta
-    fi
+    ;;
+esac
 
-    IMAGE_NAME="$NETWORK"
-    NETWORK="-g $NETWORK"
-fi
-
-build_image () {
```

Comment:
We create a subroutine here, then call it immediately. Might as well just dump in the code.